kong/assets/configuration/spec.yaml

```diff
@@ -8,13 +8,13 @@ files:
   - template: instances
     options:
     - name: kong_status_url
-      required: true
       description: URL where Kong exposes its status.
       value:
         example: http://localhost:8001/status/
         type: string
     - template: instances/openmetrics
       overrides:
+        openmetrics_endpoint.required: false
```

Comment:
But it is required
https://github.com/DataDog/integrations-core/blob/e668209b4448614c2fd8fadf6467252e4c0d9c1e/kong/datadog_checks/kong/kong.py#L43